statistics/histogram.go

```diff
@@ -352,7 +353,8 @@ func (hg *Histogram) RemoveVals(valCntPairs []TopNMeta) {
 			// If the current val smaller than current bucket's lower bound, skip it.
 			cmpResult = bytes.Compare(hg.Bounds.Column(0).GetRaw(bktIdx*2), valCntPairs[pairIdx].Encoded)
 			if cmpResult > 0 {
-				continue
+				pairIdx++
+				break
```

Comment:
It should be `continue`. We move to next pair. The bucket doesn't need to move currently.

Reply:
oh, my mistake...

Reply:
updated.